docs/manual/java/gettingstarted/GettingStartedSbt.md

```diff
@@ -12,7 +12,7 @@ Follow these steps to create your first Lagom build:
 
 ## Create a new Lagom build
 
-From the command line, invoke `sbt new` and specify the Lagom Giter8 template:
+Open a terminal or command window with administrative privileges. From the command line, invoke `sbt new` and specify the Lagom Giter8 template:
```

Comment:
That was removed in an earlier version, a mistake on my part. I'll fix again. While I'm changing, I'll also add the -Dsbt.version=0.13.13 in both java and scala. In addition, from the usability studies, I noticed that the procedures didn't specify to create a directory for the project, so I'm adding that too.